test/FsAutoComplete.Tests.Lsp/TestCases/PreviewScriptFeatures/Script.fsx

```diff
@@ -1,15 +1,13 @@
 (*
- This file demonstrates the use of F# 4.7 preview language features, 
- which must be opted-in to.  Parsing/typechecking will fail if preview
- feature are not enabled.
+ This file demonstrates the use of F# 5 language features.
 *)
 
 open System
 
-let add x y = 
+let add x y =
     if x < 0 then raise (ArgumentOutOfRangeException(nameof(x)))
     x + y
 
-open System.Math
+open type System.Math
```

Comment:
previous syntax was wrong, so this needed to happen to make the test pass after being reenabled